HW0/P5/P5_6.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
This was only to solve the part where you hand out 1 or 2 bags per second, so at the very least the solution requires N or N/2 seconds to just hand out the bags. The comment at the top is misleading, it wouldn't be optimal if you were handing out say 64 bags per second because you would then prefer a more tree-like approach that gets to log_2(N)